packages/loading-indicator/README.md

```diff
@@ -0,0 +1,46 @@
+# Loading Indicator
+
+![npm (scoped)](https://img.shields.io/npm/v/@leafygreen-ui/loading-indicator.svg)
+
+#### [View on MongoDB.design](https://www.mongodb.design/component/loading-indicator/example/)
+
+## Installation
+
+### Yarn
+
+```shell
+yarn add @leafygreen-ui/loading-indicator
+```
+
+### NPM
+
+```shell
+npm install @leafygreen-ui/loading-indicator
+```
+
+## Spinner
+
+```
+<Spinner
+  variant="large"
+  description=”Loading…”
+/>
+```
+
+| Prop           | Type      | Description                                                                        | Default   |
+| -------------- | --------- | ---------------------------------------------------------------------------------- | --------- |
+| `variant`      | `Variant` | Determines the size or orientation of the spinner and description text             | `default` |
```

Comment:
Can we confirm team alignment on this one

Reply:
It's a little confusing since it says variant, but the description says size or orientation. It also seems inconsistent with our other components.